### PR TITLE
Fix wal2json flakiness

### PR DESCRIPTION
**Pull request by chadwhitacre**
Kludge to fix https://github.com/getsentry/self-hosted/issues/1899.

### CODE REVIEW

File: install/install-wal2json.sh [hunk 1]
```diff
@@ -16,11 +16,11 @@ docker_curl() {
 }
 
 if [[ $WAL2JSON_VERSION == "latest" ]]; then
-  VERSION=$(
-    docker_curl https://api.github.com/repos/getsentry/wal2json/releases/latest |
-      grep '"tag_name":' |
-      sed -E 's/.*"([^"]+)".*/\1/'
-  )
+  docker_curl https://api.github.com/repos/getsentry/wal2json/releases/latest >wal2json.json
+  cat "====================================================="
```
Comment (hubertdeng123) [inline install/install-wal2json.sh:20]: seems wrong here?

Comment (hubertdeng123) [reply install/install-wal2json.sh:20]: also failing in e2e-tests

Comment (chadwhitacre) [reply install/install-wal2json.sh]: 😬

**Review by hubertdeng123 — APPROVED** [review]

**Comment by hubertdeng123** [comment]
oh wow, were you able to find the issue on a build failure on the PR? :O

**Comment by chadwhitacre** [comment]
Yeah, posted [up here](https://github.com/getsentry/self-hosted/issues/1899#issuecomment-1376305584).

**Comment by chadwhitacre** [comment]
Checked with SNS.

>> How likely is it that [wal2json will ever see another release](https://github.com/getsentry/wal2json/releases)?
>> Trying to decide how to handle updates in self-hosted (current approach is leading to API rate limits in CI, wondering if we can cheat).
>
> Our fork or the origin one ?
> Our fork is unlikely.
> The origin does.
> Now the question is whether we will keep doing change data capture at all.
> The two use cases are
> - Data pipelines. No self hosted. The data team wants to get rid of it, but the solution currently investigated does not work. So it is an open question.
> - Issues platform. The implementation of issue search based on CDC data was never rolled out and the team was repurposed, though there are possible ways issue platform may need it again. So this one is an open question as well

**Pull request merged**